[nativert] session state

dolpm opened the pull request:
Summary: att

Test Plan:
ci

Rollback Plan:

Differential Revision: D76827309

zhxchen17 reviewed (approved):
@dolpm plz fix linters

dolpm commented:
> @dolpm plz fix linters

did, waiting for sync.

dolpm committed:
[nativert] session state (#156190)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/156190

att

Test Plan:
ci

Rollback Plan:

Reviewed By: zhxchen17

Differential Revision: D76827309
